### DISCUSSION TITLE
Windows Server for ARM Preview reboots when booting installer

### DISCUSSION THREAD

**Discussion by Xpl0itU** [utmapp/UTM#6368] [Windows] [answered]
**Describe the issue**  
I downloaded Windows Server for ARM using CrystalFetch, however, when trying to install it, it results on a reset of the VM, and you can see the blue background and can move the cursor briefly before the reset.

Tried with builds 26080.1 and 26212.5000

**Configuration**  
* UTM Version: 4.5.2
* macOS Version: 14.4.1
* Mac Chip (Intel, M1, ...): M1 Pro

**Debug log**  
[debug.log](https://github.com/utmapp/UTM/files/15295687/debug.log)

**Comment by jrscrs** [comment]
I had the same issue when creating the ISO image with Linux. Read online that the windows variant did work, so tried creating the ISO from my windows VM and voila, it boots.

**Reply by AlfredAR8** [reply]
You know which variant works? can you share it?

**Comment by Xpl0itU** [comment, marked as answer]
Fixed by getting a Windows 11 installer and replacing the install.wim file from the windows server installer or manually deploying windows server from the windows 11 installer

**Reply by AlfredAR8** [reply]
Is there a way i can contact you? I need some help, you got discord or other contact info?

**Reply by Xpl0itU** [reply]
Yeah, discord: dathinkingchair
